---
layout: single
title: "Publications"
permalink: /publications/
author_profile: false
---

A more up to date List of publications can be found here: [Google Scholar](https://scholar.google.com/citations?user=65FxKNgAAAAJ&hl=en)

### 2025
- **Nicola F. Müller, Ryan R. Wick, Louise M. Judd, Deborah A. Williamson, Trevor Bedford, Benjamin P. Howden, Sebastián Duchêne\*, Danielle J. Ingle\***
  "Quantifying plasmid movement in drug-resistant Shigella species using phylodynamic inference"
  BioRxiv.
  [https://doi.org/10.1101/2022.10.27.514108](https://doi.org/10.1101/2022.10.27.514108)


- **Miguel I Paredes, Citina Liang, Sze-Chuan Suen, Ian W Holloway, Jacob M Garrigues, Nicole M Green, Trevor Bedford, Nicola F Mueller\*, Joesph Osmundson\***
  "Viral introductions and return to baseline sexual behaviors maintain low-level mpox incidence in Los Angeles County, USA, 2023-2024"
  BioRxiv.
  [https://doi.org/10.1101/2025.03.14.25323999](https://doi.org/10.1101/2025.03.14.25323999)


### 2024

- **M. I. Paredes, N. Ahmed, M. Figgins, V. Colizza, P. Lemey, J. T. McCrone, N. F. Müller, C. Tran-Kiem, T. Bedford**
  "Underdetected dispersal and extensive local transmission drove the 2022 mpox epidemic"<br />
  *Cell* 1374-1386. e13
  [https://doi.org/10.1016/j.cell.2024.02.003](https://doi.org/10.1016/j.cell.2024.02.003)

- **N. Walas, N. F. Müller, E. Parker, A. Henderson, D. Capone, J. Brown, T. Barker, J. P. Graham**
  “Phylodynamics Uncovers the Transmission of Antibiotic-Resistant Escherichia coli between Canines and Humans in an Urban Environment.”<br />
  *Science of The Total Environment* 170139
  [https://doi.org/10.1016/j.scitotenv.2024.170139](https://doi.org/10.1016/j.scitotenv.2024.170139)


- **M. I. Paredes, A. C. Perofsky, C. Viboud, H. Chu, N. F. Müller\*, T. Bedford\***
  "Local-scale phylodynamics reveal differential community impact of SARS-CoV-2 in a metropolitan US county.”<br />
  *PLoS pathogens* e1012117
  [https://doi.org/10.1371/journal.ppat.1012117](https://doi.org/10.1371/journal.ppat.1012117)


### 2023
- **M. A. Lewinsohn, T. Bedford, N. F. Müller\*, A. F. Feder\***  
  “State-dependent evolutionary models reveal modes of solid tumor growth”  
  Accepted for publication in *Nat Ecol Evol* 7, 581–596 (2023).  
  [https://doi.org/10.1038/s41559-023-02000-4](https://doi.org/10.1038/s41559-023-02000-4)

- **M. Layan, N. F. Müller, S. Dellicour, N. De Maio, H. Bourhy, S. Cauchemez, G. Baele**  
  “Impact and mitigation of sampling bias to determine viral spread: evaluating discrete phylogeography through CTMC modeling and structured coalescent model approximations”  
  *Virus Evolution*, vead010  
  [https://doi.org/10.1093/ve/vead010](https://doi.org/10.1093/ve/vead010)

### 2022
- **N. F. Müller, K. Kistler, T. Bedford**  
  “A Bayesian approach to infer recombination patterns in coronaviruses."  
  *Nature Communications* 13, 4186.  
  [https://doi.org/10.1038/s41467-022-31749-8](https://doi.org/10.1038/s41467-022-31749-8)

- **D. Chakraborty, C. Guinat, N. F. Müller, F. Briand, M. Andraud, A. Scoizec, S. Lebouquin, E. Niqueux, A. Schmitz, B. Grasland, J. Guerin, M. C. Paul, T. Vergne**  
  “Phylodynamic assessment of control measures for highly pathogenic avian influenza epidemics in France.”  
  *Transboundary and emerging diseases.*  
  [https://doi.org/10.1111/tbed.14490](https://doi.org/10.1111/tbed.14490)

- **U. Stolz, T. Stadler, N. F. Müller\*, T. G. Vaughan\***  
  “Joint inference of migration and reassortment patterns for viruses with segment genomes”  
  *Molecular Biology and Evolution*, msab342.  
  [https://doi.org/10.1093/molbev/msab342](https://doi.org/10.1093/molbev/msab342)

### 2021
- **D. C. Owuor, Z. R. de Laurent, G. K. Kikwai, L. M. Mayieka, M. Ochieng, N. F. Müller, N. A. Otieno, E. N. Hunsperger, R. Garten, J. R. Barnes, S. S. Chaves, J. D. Nokes, C. N. Agoti**  
  “Characterizing the countrywide epidemic spread of influenza A(H1N1)pdm09 virus in Kenya between 2009 and 2018.”  
  *Viruses* 13, no. 10: 1956.  
  [https://doi.org/10.3390/v13101956](https://doi.org/10.3390/v13101956)

- **N. F. Müller\*, C. Wagner\*, C. D. Frazar\*, P. Roychoudhury\*, A. L. Greninger, D. A. Nickerson, T. Bedford**  
  “Viral genomes reveal patterns of the SARS-CoV-2 outbreak in Washington State.”  
  *Science Translational Medicine* 2021 May 26;13(595).  
  [https://doi.org/10.1126/scitranslmed.abf0202](https://doi.org/10.1126/scitranslmed.abf0202)

- **L. H. Moncla, A. Black, C. DeBolt, M. Lang, N. R. Graff, A. C. Pérez-Osorio, N. F. Müller, D. Haselow, S. Lindquist, T. Bedford**  
  "Repeated introductions and intensive community transmission fueled a mumps virus outbreak in Washington State."  
  *Elife* 10 (2021): e66448.  
  [https://doi.org/10.7554/eLife.66448](https://doi.org/10.7554/eLife.66448)

- **N. F. Müller, H. Ogilvie, C. Zhang, A. Drummond, T. Stadler**  
  “Inference of species histories in the presence of gene flow.”  
  *BioRxiv*  
  [https://doi.org/10.1101/348391](https://doi.org/10.1101/348391)

### 2020
- **N. F. Müller, D. Wüthrich, H. M. B. Seth-Smith, Y. Hollenstein, M. Syedbasha, D. Lang, R. A. Neher, O. Dubuis, M. Naegele, A. Buser, C. H. Nickel, N. Ritz, A. Zeller, B. M. Lang, J. Hadfield, T. Bedford, M. Battegay, R. Schneider-Sliwa, A. Egli, T. Stadler**  
  “Characterising the epidemic spread of Influenza A/H3N2 within a city through phylogenetics”  
  *PLOS Pathogens* 16(11): e1008984.  
  [https://doi.org/10.1371/journal.ppat.1008984](https://doi.org/10.1371/journal.ppat.1008984)

- **N. F. Müller, R. Bouckaert**  
  “Adaptive Metropolis-coupled MCMC for BEAST 2”  
  *PeerJ* 8:e9473.  
  [https://doi.org/10.7717/peerj.9473](https://doi.org/10.7717/peerj.9473)

- **N. F. Müller, U. Stolz, G. Dudas, T. Stadler, T. G. Vaughan**  
  “Bayesian inference of reassortment networks reveals fitness benefits of reassortment in human influenza viruses”  
  *Proceedings of the National Academy of Sciences* Jul 2020, 117 (29) 17104-17111;  
  [https://doi.org/10.1073/pnas.1918304117](https://doi.org/10.1073/pnas.1918304117)

- **T. Bedford, A. L. Greninger, P. Roychoudhury, …, N. F. Müller, …, H. Y. Chu, J. Shendure, K. R. Jerome,**  
  “Cryptic transmission of SARS-CoV-2 in Washington state”,  
  *Science* 370, no. 6516 (2020): 571-575.  
  [https://doi.org/10.1126/science.abc0523](https://doi.org/10.1126/science.abc0523)

- **A. Egli, N. Goldman, N. F. Müller, M. Brunner, D. Wüthrich, S. Tschudin-Sutter, E. Hodcroft, R. Neher, C. Saalfrank, J. Hadfield, T. Bedford, M. Syedbasha, T. Vogel, N. Augustin, J. Bauer, N. Sailer, N. Amar-Sliwa, D. Lang, H. M.B. Seth-Smith, A. Blaich, Y. Hollenstein, O. Dubuis, M. Nägele, A. Buser, C. H. Nickel, N. Ritz, A. Zeller, T. Stadler, M. Battegay, R. Schneider-Sliwa**  
  “High-resolution influenza mapping of a city reveals socioeconomic determinants of transmission within and between urban quarters.”  
  *BioRxiv*  
  [https://doi.org/10.1101/2020.04.03.023135](https://doi.org/10.1101/2020.04.03.023135)

### 2019
- **N. F. Müller, G. Dudas, & T. Stadler**  
  “Inferring time-dependent migration and coalescence patterns from genetic sequence and predictor data in structured populations”  
  *Virus Evolution*.  
  [https://doi.org/10.1093/ve/vez030](https://doi.org/10.1093/ve/vez030)

- **J. Yang, N. F. Müller, R. Bouckaert, B. Xu & A. J. Drummond**  
  “Bayesian phylodynamics of avian influenza virus H9N2 in Asia with time-dependent predictors of migration”  
  *PLoS Computational Biology*.  
  [https://doi.org/10.1371/journal.pcbi.10071](https://doi.org/10.1371/journal.pcbi.10071)

- **M. Zampieri, M. Hörl, F. Hotz, N. F. Müller, U. Sauer;**  
  “Regulatory mechanisms underlying coordination of amino acid and glucose catabolism in Escherichia coli”  
  *Nature Communications*.  
  [https://doi.org/10.1038/s41467-019-11331-5](https://doi.org/10.1038/s41467-019-11331-5)

- **A. Egli, C. Saalfrank, N. Goldman, M. Brunner, Y. Hollenstein, T. Vogel, N. Augustin, D. Wüthrich, H.M.B. Seth-Smith, E. Roth, M. Syedbasha, N. F. Müller, D. Vogt, J. Bauer, N. Amar-Sliwa, D. M. Meinel, O. Dubuis, M. Naegele, S. Tschudin-Sutter, A. Buser, C. H. Nickel, A. Zeller, N. Ritz, M. Battegay, T. Stadler, R. Schneider-Sliwa**  
  “Identification of influenza urban transmission patterns by geographical, epidemiological and whole genome sequencing data: protocol for an observational study.”  
  *BMJ open*, 9(8), p.e030913.  
  [https://doi.org/10.1136/bmjopen-2019-030913](https://doi.org/10.1136/bmjopen-2019-030913)

### 2018
- **N. F. Müller, D. A. Rasmussen, T. Stadler**  
  “MASCOT: parameter and state inference under the marginal structured coalescent approximation.”  
  *Bioinformatics*, bty406.  
  [https://doi.org/10.1093/bioinformatics/bty406](https://doi.org/10.1093/bioinformatics/bty406)

- **J. Barido-Sottani, V. Boskova, L. du Plessis, D. Kühnert, C. Magnus, V. Mitov, N. F. Müller, J. Pecerska, D. Rasmussen, C. Zhang, A. J. Drummond, T. Heath, O. Pybus, T. Vaughan, T. Stadler**  
  “Taming the BEAST—A Community Teaching Material Resource for BEAST 2.”  
  *Systematic Biology*, Volume 67, Issue 1, Pages 170–174.  
  [https://doi.org/10.1093/sysbio/syx060](https://doi.org/10.1093/sysbio/syx060)

### 2017
- **N. F. Müller, D. A. Rasmussen, T. Stadler**  
  “The Structured Coalescent and its Approximations.”  
  *Molecular Biology and Evolution*, Volume 34, Issue 11, Pages 2970–2981.  
  [https://doi.org/10.1093/molbev/msx186](https://doi.org/10.1093/molbev/msx186)
