---
layout: single
title: "Research"
permalink: /research/
author_profile: false
---

## Overview

We seek to use genomic epidemiology to quantify transmission dynamics of bacterial and viral infectious diseases with the ultimate goal of informing public health interventions. 

**Local Phylodynamics to inform public health interventions**


After its emergence, and despite travel restrictions SARS-CoV-2 spread rapidly around the world in early
2020.
Similarly, influenza viruses move between regions, causing seasonal outbreaks in the northern and
southern hemispheres, despite often going almost locally extinct between the seasons. While this global
spread starts novel outbreaks, the majority of cases result from local transmission, such as within households,
schools, workplaces, public transportation, etc. We aim to develop computational approaches to infer local transmission dynamics of infectious diseases jointly from multiple data sources. Using this we will investigate how we can utilize different sources of data collected as part of routine infectious diseases surveillance to quantify i) the
relative disease burden of different neighborhoods and ii) the rates of transmission between different
neighborhoods. 

<div style="text-align: center;">
    <img src="/assets/images/Dataflow.png" alt="localphylo" style="width: 100%; height: auto; margin: 0 auto;">
</div>

<br>

**Genomic epidemiology to quantify the spread of bacterial poathogen:**.

Investigating the emergence and spread of antimicrobial resistance Antibiotics have dramatically reduced the burden of bacterial pathogens on human health. Antimicrobial resistance threatens these achievements. Antimicrobial resistance can be acquired de novo or through horizontal gene transfer. Resistance genes can be spread, for example, between hospital and community settings or between humans and domesticated animals. How exactly this spread unfolds is, however, unclear and quantifying the relative contributions of different places or host types to the spread of antibiotic resistance is crucial to design efficient interventions to reduce spread of resistance. Genomic sequences offer a pathway to study these patterns.
<div style="text-align: center;">
    <img src="/assets/images/scheme.png" alt="bacterial phylo" style="width: 100%; height: auto; margin: 0 auto;">
</div>

<br>

**How does recombination shape the evolution and transmission of viruses:**

Reassortment, the reshuffling of the different segments of influenza viruses, is the only documented natural cause for the emergence of novel pandemic influenza viruses. On top of promoting the ability to switch between host species, intra-subtype reassortment can impact viral fitness by combining segments from different viral lineages. Reconstructing past reassortment events using phylogenetics, allows us to infer which reassortment. Such population level spread is driven by factors beyond cellular processes, such as potential fitness benefits for transmission between hosts. Better understanding the role of reassortment process in creating novel emerging viruses could help identify situations that promote host jumps. 

<div style="text-align: center;">
    <img src="/assets/images/H3N2.png" alt="bacterial phylo" style="width: 60%; height: auto; margin: 0 auto;">
</div>


