---
layout: single
title: "Research"
permalink: /research/
author_profile: false
---

## Overview

We seek to quantify transmission dynamics of bacterial and viral infectious diseases from pathogen genome sequence data with the ultimate goal of informing public health interventions to reduce disease burden. 
Methodologically, we use phylodynamic methods to infer transmission dynamics from pathogen genome sequence data. 
We are particularly interested in developing methods to infer transmission dynamics from multiple data sources, such as pathogen genome sequences, epidemiological data, and spatial data. 
We apply these methods to study the transmission dynamics of influenza viruses, SARS-CoV-2, and bacterial pathogens. 

An incomplete list of research interests include:

**Local Phylodynamics to inform public health interventions**

Phylodynamics is typically used to infer transmission dynamics of infectious diseases at the global scale.
While this global spread starts novel outbreaks, the majority of cases result from local transmission, such as within households, schools, workplaces, public transportation, etc. 
We aim to develop computational approaches to infer local transmission dynamics of infectious diseases jointly from multiple data sources. 
Using this we will investigate how we can utilize different sources of data collected as part of routine infectious diseases surveillance to quantify i) the relative disease burden of different neighborhoods and ii) the rates of transmission between different neighborhoods. 

<div style="text-align: center;">
    <img src="/assets/images/Dataflow.png" alt="localphylo" style="width: 100%; height: auto; margin: 0 auto;">
</div>

<br>

**Genomic epidemiology to quantify the spread of bacterial pathogens:**

Antibiotics have dramatically reduced the burden of bacterial pathogens on human health. 
Antimicrobial resistance threatens these achievements. 
Antimicrobial resistance can be acquired de novo or through horizontal gene transfer. R
esistance genes can be spread, for example, between hospital and community settings or between humans and domesticated animals. 
How exactly this spread unfolds is, however, unclear and quantifying the relative contributions of different places or host types to the spread of antibiotic resistance is crucial to design efficient interventions to reduce spread of resistance. 
Our goal is to develop phylodynamics approaches to quantify the spread of antibiotic resistance genes between different host types and geographic areas.

<div style="text-align: center;">
    <img src="/assets/images/scheme.png" alt="bacterial phylo" style="width: 100%; height: auto; margin: 0 auto;">
</div>

<br>

**How recombination shapes the evolution and transmission of viruses:**

Reassortment, the reshuffling of the different segments of influenza viruses, is the only documented natural cause for the emergence of novel pandemic influenza viruses. On top of promoting the ability to switch between host species, intra-subtype reassortment can impact viral fitness by combining segments from different viral lineages. Reconstructing past reassortment events using phylogenetics, allows us to infer which reassortment. Such population level spread is driven by factors beyond cellular processes, such as potential fitness benefits for transmission between hosts. 
We seek to elucidate the role of reassortment in the evolution and transmission of influenza viruses.

<div style="text-align: center;">
    <img src="/assets/images/H3N2.png" alt="bacterial phylo" style="width: 100%; height: auto; margin: 0 auto;">
</div>


